ex36.py

```diff
@@ -280,6 +291,9 @@ def skeleton(current_count):
 You are aware you are not alone.
 Four undead join you in the dungeon.
 Well, %i colourful skeletons to be exact.
+
+# insert skeleton here
+
 \n(Press enter to continue)""" % len(skeleton_list)
```

Comment:
One nice feature of Python is the more modern format() approach to strings.
e.g.
```
'Well, {number_of} colourful skeletons to be exact'.format(number_of=len(skeleton_list))
```